knexfile-tables.js

```diff
@@ -0,0 +1,23 @@
+const env = require('./.env');
```

Comment:
The ```dotenv``` package is the standard solution to use ```.env``` files. There is no need for the content of the file to be in the json format (this applies to all files in the PR).